safeeyes/core.py

```diff
@@ -99,9 +90,7 @@ def start(self, next_break_time=-1, reset_breaks=False):
                 utility.start_thread(self.__scheduler_job)
 
     def stop(self, is_resting=False):
-        """
-        Stop Safe Eyes if it is running.
-        """
+        """Stop Safe Eyes if it is running."""
```

Comment:
In my opinion, such docstrings are not helpful and could be removed or partially placed into method name. This could be a point in our code styleguide

Reply:
I agree. Are there linter rules that we could use that would help with that?
As with the other ones, I would leave this for a future PR as well.